src/ops/info.rs

```diff
@@ -49,6 +50,7 @@ pub fn info(
 
     validate_locked_and_frozen_options(package_id, config)?;
 
+    let current_rustc_version = &config.load_global_rustc(ws.as_ref())?.version;
```

Comment:
#69 says to fallback to `rustc --version` if either
- We don't have a parent manifest
- The parent manifest doesn't set `package.rust-version`